agent/tool-scripts/postprocess/openshift-metrics-postprocess

```diff
@@ -6,26 +6,23 @@ input_file = sys.argv[1]
 output_file = sys.argv[2]
 in_csv = []
 metrics=[]
-
-with open(input_file, 'r') as f:
-    reader = csv.reader(f)
+with open(input_file, 'r') as file:
```

Comment:
Here we are going to want to re-write this to process the JSON from the `prom2json` tool, changing the field names as well.